### DISCUSSION TITLE
More development

### DISCUSSION THREAD

**Discussion by benharbit** [yessGlory17/nft-marketplace#49] [General]
Is there anyone interested in developing this further?

**Comment by yessGlory17** [comment]
I'll be back to developing again next month :tada:

**Reply by zergormagn** [reply]
needs some major cleanup, dependencies don't resolve on clean install. Seems quite bloated with the extensive libraries are they all needed? could also use minimum server requirements recommendation, and a demo website?

**Reply by benharbit** [reply]
I added Typescript to fork and connected it to a NFT contract I deployed on Goerli.    Maybe I will setup a demo website

**Reply by benharbit** [reply]
Also I added a live demo website here https://nft-marketplace.herokuapp.com/
